src/components/message-pane/message-item/MessageItem.tsx

```
@@ -1401,10 +1401,10 @@ const FileEmbed = (props: {
           />
           {props.error
             ? t("fileEmbed.expired", {
-                time: timeSinceMentions(props.file?.expireAt!)
+                time: formatTimestampRelative(props.file?.expireAt!)
```

Biome (2.4.4)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)